client/src/components/EditProfile/AvailabilityTab/AvailabilityRow/AvailabilityRow.tsx

```diff
@@ -0,0 +1,96 @@
+import useStyles from './useStyles';
+import { Box, Typography, Grid, Select, MenuItem } from '@material-ui/core';
+import { useFormik } from 'formik';
+
+const times = [...Array(24).keys()].map((number) => {
```

Comment:
I think we're using this function in the 'Edit Profile' form too? In that case you can move this function to a util folder and reuse it rather than redefining it :)